etc/restore-old-bindings.conf

```diff
@@ -18,16 +18,11 @@
 
 # changed in mpv 0.26.0
 
-h cycle tv-channel -1                  # previous channel
-k cycle tv-channel +1                  # next channel
-H cycle dvb-channel-name -1            # previous channel
-K cycle dvb-channel-name +1            # next channel
```

Comment:
perhaps keep these around with a note that the old behavior is no longer available

Reply:
Alternatively, we could also replace the DVB bindings with their successors:
```
H cycle dvbin-channel-switch-offset up
K cycle dvbin-channel-switch-offset down
```
(see https://github.com/mpv-player/mpv/pull/7010 ).